scripts/apps/authoring/authoring/helpers.js

```diff
@@ -231,7 +231,7 @@ export function cutoffPreviousRenditions(update, origItem) {
         try {
             updateRenditions = update.associations[key].renditions;
             origRenditions = origItem.associations[key].renditions;
-            if (!(updateRenditions && origRenditions)) {
+            if (!updateRenditions || !origRenditions) {
```

Comment:
Can you use `if(updateRenditions == null || origRenditions == null)` instead of `if (!updateRenditions || !origRenditions)`?

Reply:
It's value is sometimes undefined and sometimes null.

Reply:
Double equals covers for it.
![screenshot](https://user-images.githubusercontent.com/2076953/51603823-2d806000-1f0b-11e9-9049-2d9a79b09cb4.png)

Reply:
done